i18n-helpers/src/bin/mdbook-gettext.rs

```diff
@@ -66,19 +68,30 @@ fn translate(text: &str, catalog: &Catalog) -> String {
 }
 
 fn translate_book(ctx: &PreprocessorContext, mut book: Book) -> anyhow::Result<Book> {
+    // no-op when the target language is not set
+    if ctx.config.book.language.is_none() {
+        return Ok(book);
+    }
+
+    // the target language
+    let language = ctx.config.book.language.as_ref().unwrap();
+
+    // Find PO file for the target language
     let cfg = ctx
         .config
         .get_preprocessor("gettext")
         .ok_or_else(|| anyhow!("Could not read preprocessor.gettext configuration"))?;
-    let path = cfg
-        .get("po-file")
-        .ok_or_else(|| anyhow!("Missing preprocessor.gettext.po-file config value"))?
-        .as_str()
-        .ok_or_else(|| anyhow!("Expected a string for preprocessor.gettext.po-file"))?;
-    let catalog = po_file::parse(Path::new(path))
-        .map_err(|err| anyhow!("{err}"))
-        .with_context(|| format!("Could not parse {path} as PO file"))?;
+    let po_dir = cfg.get("po-dir").and_then(Value::as_str).unwrap_or("po");
+    let path = ctx.root.join(po_dir).join(format!("{language}.po"));
 
+    // no-op when PO file is missing
```

Comment:
Would it not be clearer to error out if the file doesn't exist? I'm thinking that the normal case is to have `book.language` unset, so _if_ it is set, it means that the user has actively asked for a non-standard language. This in turn makes me think that the user would like to be told about a missing PO file.

Reply:
It's for the English (the original book), which requires no translation but sets `book.language = en`. 

I was a bit hesitant as well for this _fallback_. Probably we can add `skip-for-languages = ["en"]` kind of setting.

Reply:
And I was thinking additional "language specific override" like following:
```
[preprocessor.gettext]
en.skip = true
ko.po-file = "path-to-ko.po"
```

Reply:
> It's for the English (the original book), which requires no translation but sets `book.language = en`.

Ah... I had forgotten that we set the language even in that case.

> And I was thinking additional "language specific override" like following:
> 
> ```
> [preprocessor.gettext]
> en.skip = true
> ko.po-file = "path-to-ko.po"
> ```

Okay, do you have a use case for this right now?